app/features/quizzify/core.py

```diff
@@ -1,8 +1,8 @@
-from app.services.tool_registry import ToolFile
```

Comment:
You must not need to remove the app folder because it allows to avoid dependency issues.